docker.d/init.sh

```diff
@@ -31,6 +31,9 @@ case $ENV in
     ;;
   dev)
     export TRUST_LEVEL=1
+    if [ $PROJECT_NAME = "signing" ]; then
```

Comment:
was this something that stayed over from when you were testing?

Reply:
No, I think this is what we wanted to happen...but it seems to be causing a lot of confusion.

The idea was to have:
* gecko-3-signing for the official release signing. This corresponds to the "prod" environment.
* gecko-t-signing for the unofficial "dep" signing on try/autoland/etc. This corresponds to the "fake-prod" environment.
* gecko-t-signing-dev for our development work. This corresponds to the "nonprod" environment.

So this hunk of the patch gives us gecko-t-signing-dev instead of gecko-1-signing-dev.

Reply:
This is also specific to just `signing` I believe. For rest of workers, we're going to have `gecko-1-{script}-dev`. I double-checked and fixed yesterday some scope and they were all broken because of the initial proposal that we had around (which stated the use of `t` for the `level-1-dev` counterpart. So I think for signing, this is acceptable.